packages/ember-cli-code-coverage/index.js

```diff
@@ -1,85 +1,23 @@
 'use strict';
 
-var path = require('path');
-var fs = require('fs-extra');
 var attachMiddleware = require('./lib/attach-middleware');
-var config = require('./lib/config');
-const walkSync = require('walk-sync');
-const VersionChecker = require('ember-cli-version-checker');
-
-function getPlugins(appOrAddon) {
-  let options = (appOrAddon.options = appOrAddon.options || {});
-  options.babel = options.babel || {};
-  return (options.babel.plugins = options.babel.plugins || []);
-}
-
-// Regular expression to extract the file extension from a path.
-const EXT_RE = /\.[^.]+$/;
 
 module.exports = {
   name: require('./package').name,
 
-  /**
-   * Look up the file path from an ember module path.
-   * @type {Object<String, String>}
-   */
-  fileLookup: null,
-
-  // Ember Methods
-  init() {
-    this._super.init.apply(this, arguments);
-    this.fileLookup = {};
-  },
-
-  included(appOrAddon) {
-    this._super.included.apply(this, arguments);
-
-    this.parentRegistry = appOrAddon.registry;
-
-    if (!this._registeredWithBabel && this._isCoverageEnabled()) {
-      let checker = new VersionChecker(this.parent).for(
-        'ember-cli-babel',
-        'npm'
-      );
+  buildBabelPlugin(opts = {}) {
+    let project = opts.project;
+    let cwd = opts.cwd || project.root;
+    let include = opts.include || '**/*';
+    let exclude = opts.exclude || ['*/mirage/**/*'];
```

Comment:
Let's add a short circuit return here if `process.env.COVERAGE` is falsey, this brings.
```suggestion
  buildBabelPlugin(opts = {}) {
    if (process.env.COVERAGE) {
      return [];
    }

    let project = opts.project;
    let cwd = opts.cwd || project.root;
    let include = opts.include || '**/*';
    let exclude = opts.exclude || ['*/mirage/**/*'];
```